index.js

```diff
@@ -73,6 +74,25 @@ class Shortcut {
 		 * @type {string}
 		 */
 		this.id = id;
+
+	}
+
+	/**
+	 * Gets the shortcuts metadata from the downloadURL
+	 * @returns {Object} An object with all the metadata about the shortcut
```

Comment:
Do you think we could make a `ShortcutMetadata` class with a custom, more JSON-friendly format?

Reply:
Good idea!

Reply:
Not sure if the implementation I added in the latest commit makes sense. Since this is not typescript I'm unsure of how to do this correctly 😁.

Please feel free to add some feedback.